app/workers/commit_monitor_handlers/commit/bugzilla_common/bugzilla_common.rb

```diff
@@ -0,0 +1,33 @@
+module CommitMonitorHandlers
+  module Commit
+    module BugzillaCommon
+      def product
+        Settings.commit_monitor.bugzilla_product
+      end
+
+      def with_bug(bug_id)
+        return unless block_given?
+        MiqToolsServices::Bugzilla.call do
+          output = ActiveBugzilla::Bug.find(:product => product, :id => bug_id)
+          if output.empty?
+            logger.error "Unable to write for bug id #{bug_id}: Not a '#{product}' bug."
+          else
+            bug = output.first
+            yield(bug)
+          end
+        end
+      rescue => err
+        logger.error "Unable to write for bug id #{bug_id}: #{err}"
```

Comment:
For both of these cases, would it make sense to just return without a value and write a better log message?  That way the caller doesn't have to handle anything in a block that it doesn't want to be in anyway because we couldn't find the bug.